### PR TITLE
Remove current SQLite DB driver

### PR DESCRIPTION
**Pull request by mheon**
The SQLite DB backend has become an unmanageable nightmare. I like having the option for DB work, but it's become an active hindrance to further development, and it's definitely not in any shape to be actively used.

This should also drastically lower our binary size.

I'd like to find the time to rewrite this eventually using an ORM. We'll see if I find the time.

### CODE REVIEW

**Comment by mheon** [comment]
@rhatdan PTAL

**Comment by mheon** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@baude @umohnani8 @TomSweeneyRedHat PTAL

**Comment by umohnani8** [comment]
LGTM

File: libpod/runtime.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ const (
 	// reboot
 	InMemoryStateStore RuntimeStateStore = iota
 	// SQLiteStateStore is a state backed by a SQLite database
+	// It is presently disabled
```
Comment (TomSweeneyRedHat) [inline libpod/runtime.go]: Any reason to just delete this now?  I'm not finding other references in the code other than what you've removed already here.

Comment (TomSweeneyRedHat) [reply libpod/runtime.go]: oops.  just == not just.  Basically, can we just delete this now rather than commenting/no-oping it?

**Comment by baude** [comment]
nukify ... lgtm

**Pull request closed without merging**